### PR TITLE
[14.0] [mig] document_quick_access

### PR DESCRIPTION
**Pull request by fshah-initos**

### CODE REVIEW

File: document_quick_access/views/document_quick_access_rule.xml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        <field
+            name="name"
+        >document.quick.access.rule.form (in document_quick_access)</field>
+        <field name="model">document.quick.access.rule</field>
+        <field name="arch" type="xml">
+            <form>
+                <header />
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button
```
Comment (etobella) [inline document_quick_access/views/document_quick_access_rule.xml]: imagen
Change it please on the new way of 14.0

**Review by etobella — APPROVED** [review]
Works as expected. No major changes, just a small error on one view :wink:

**Review by dsolanki-initos — APPROVED** [review]
LGTM

File: document_quick_access/views/document_quick_access_rule.xml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+<odoo>
+    <record id="document_quick_access_rule_form_view" model="ir.ui.view">
+        <field
+            name="name"
+        >document.quick.access.rule.form (in document_quick_access)</field>
+        <field name="model">document.quick.access.rule</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <button
```
Comment (etobella) [inline document_quick_access/views/document_quick_access_rule.xml]: The standard is a ribbon, not a button

Comment (fshah-initos) [reply document_quick_access/views/document_quick_access_rule.xml]: If we use the ribbon then we can't restore it back. so I have used the button.

Comment (etobella) [reply document_quick_access/views/document_quick_access_rule.xml]: You can, use More Actions

**Comment by etobella** [comment]
Review pre-commit please

**Comment by schhatbar-initos** [comment]
Is it possible to merge this PR?

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**